src/locales/en/translation.json

```diff
@@ -656,6 +657,7 @@
   "rofl": {
     "active": "Active",
     "inactive": "Inactive",
+    "rak": "RAK",
```

Comment:
Can this be "Runtime Attestation Key", or do we need to save space? (and is this even Runtime Attestation Key)

Reply:
It's "RAK" because we are using this abbreviation in ROFL app detail page (but in table context, where space matter). Anyway wanted to use the same label across different places. Do we have place to show full name in events details - yes. 

I am waiting for backend feedback too.